### PR TITLE
:page_facing_up: Update Documentation for License on Model Weights

### PR DESCRIPTION
**Pull request by simongraham**
Code is held under a BSD-3-Clause license, but model weights are dependent on the data it is trained on. Before, we generalised this and set everything under a non-commercial creative commons license, but this is not reasonable for models trained on permissive dataset, such as [PCam](https://github.com/basveeling/pcam).

This PCam specified a license per model in accordance with the original dataset license.

Documentation available at this link https://tia-toolbox.readthedocs.io/en/doc-update-model-licenses/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,9 +127,9 @@ or
 
 ### License
 
-The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [The 3-Clause BSD License].
+The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [BSD-3-Clause license](https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/LICENSE). The full text of the licence is included in [LICENSE](https://raw.githubusercontent.com/TissueImageAnalytics/tiatoolbox/develop/LICENSE).
```
Comment (shaneahmed) [inline README.md:130]:
```suggestion
The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [BSD-3-Clause license](https://github.com/TissueImageAnalytics/tiatoolbox/blob/master/LICENSE). The full text of the licence is included in [LICENSE](https://tia-toolbox.readthedocs.io/en/latest/license.html).
```

File: README.md [hunk 2]
```diff
@@ -127,9 +127,9 @@ or
...
 
-The full text of the licence is included in [LICENSE](https://raw.githubusercontent.com/TissueImageAnalytics/tiatoolbox/develop/LICENSE).
+Models weights are dependent on the datasets that they were trained on. Refer to the [documentation](https://tia-toolbox.readthedocs.io/en/latest/index.html) for more details.
```
Comment (shaneahmed) [inline README.md:133]: - [ ] Please also add this link https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE to https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE

- [ ] Please change line 25 to <a href="#license">

File: docs/license.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
+.. _license-page:
+
+License Information
+===================
+
+Source Code
+^^^^^^^^^^^
+
+All model code is held under the `BSD-3 Clause License <https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/LICENSE>`_.
```
Comment (shaneahmed) [inline docs/license.rst:9]:
```suggestion
All model source code is held under the `BSD-3 Clause License <https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/LICENSE>`_.
```

File: docs/license.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
+.. _license-page:
```
Comment (shaneahmed) [inline docs/license.rst:1]: This file has copied most of the text from pretrained models. Why not just update pretrained models page?

File: docs/license.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
```
Comment (shaneahmed) [inline docs/license.rst:2]: you will also need to add this to index.html to appear as a tab on readthedocs.

**Review by shaneahmed — CHANGES_REQUESTED** [review]
Thanks @simongraham I have added my comments.

File: docs/license.rst (hunk 4 above)
Comment (simongraham) [reply docs/license.rst]: Good point - I have now simplified the PR and updated `pretrained.rst` instead

File: docs/license.rst (hunk 5 above)
Comment (simongraham) [reply docs/license.rst]: Not using `license.rst` anymore

File: docs/license.rst (hunk 3 above)
Comment (simongraham) [reply docs/license.rst]: Not using `license.rst` anymore, but I have put this snippet at the top of `pretrained.rst` with the suggested change.

File: README.md (hunk 2 above)
Comment (simongraham) [reply README.md:133]: The first comment I don't understand - both links are the same.

Comment (shaneahmed) [reply README.md:133]: sorry, please add this link https://tia-toolbox.readthedocs.io/en/latest/pretrained.html to https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE. To clarify that model weights are under a different license.

You can change the https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE file by adding this to the top.
```
BSD 3-Clause License

Model weights are provided under a different license. For details, please see https://tia-toolbox.readthedocs.io/en/latest/pretrained.html

Copyright (c) 2022, Tissue Image Analytics (TIA) Centre
All rights reserved.
```

File: README.md [hunk 6]
```diff
@@ -127,9 +127,9 @@ or
 
 ### License
 
-The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [The 3-Clause BSD License].
+The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [BSD-3-Clause license](https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/LICENSE). The full text of the licence is included in [LICENSE](https://raw.githubusercontent.com/TissueImageAnalytics/tiatoolbox/develop/LICENSE).
 
-The full text of the licence is included in [LICENSE](https://raw.githubusercontent.com/TissueImageAnalytics/tiatoolbox/develop/LICENSE).
+Models weights are dependent on the datasets that they were trained on. Refer to the [documentation](https://tia-toolbox.readthedocs.io/en/latest/pretrained.html) for more details.
```
Comment (shaneahmed) [inline README.md:132]:
```suggestion
Models weights are dependent on the datasets that they were trained on. Please refer to the [documentation](https://tia-toolbox.readthedocs.io/en/latest/pretrained.html) for more details.
```

**Review by shaneahmed — APPROVED** [review]
Thanks @simongraham

**Pull request merged**